pkg/mcs/scheduling/server/apis/v1/api.go

```diff
@@ -121,6 +121,9 @@ func NewService(srv *scheserver.Service) *Service {
 func (s *Service) RegisterAdminRouter() {
 	router := s.root.Group("admin")
 	router.PUT("/log", changeLogLevel)
+	cacheRouter := router.Group("cache/regions")
```

Comment:
How about only cache or just not create a new group?

Reply:
Do you mean `cacheRouter := s.root.Group("admin/cache/regions")`

Reply:
cacheRouter := s.root.Group("cache") or just use router.DELETE("cache/regions", xxx)